templates/versafix-1/template-versafix-1.html

```diff
@@ -294,7 +294,7 @@
         /* buttons */
         buttonStyle color { label: Text Color; extend: color }
         buttonStyle size { label: Text Size; extend: size }
-        buttonStyle { label: Button; properties: face color size buttonColor radius paddingTop paddingBottom borderWidth borderType borderColor }
+        buttonStyle { label: Button; properties: align face color size buttonColor radius paddingTop paddingBottom borderWidth borderType borderColor }
```

Comment:
this way it will appear on all buttons but will it work then for all blocks with buttons?